releasenotes/notes/deprecate-condition_c_if-9548e5522814fe9c.yaml

```diff
@@ -0,0 +1,38 @@
+---
+deprecations_circuits:
+  - |
+    Deprecated the :attr:`.Instruction.condition` attribute and the
+    :meth:`.Instruction.c_if` method. They will be removed
+    in Qiskit 2.0, along with any uses in the Qiskit data
+    model. This functionality has been superseded by the :class:`.IfElseOp` class
+    which can be used to describe a classical condition in a circuit. For
+    example, a circuit using  :meth:`.Instruction.c_if` like::
+
+        from qiskit.circuit import QuantumCircuit
+
+        qc = QuantumCircuit(2, 2)
+        qc.h(0)
+        qc.x(0).c_if(0, 1)
+        qc.z(1.c_if(1, 0)
+        qc.measure(0, 0)
+        qc.measure(1, 1)
+
+    can be rewritten as::
+
+        qc = QuantumCircuit(2, 2)
+        qc.h(0)
+        with expected.if_test((expected.clbits[0], True)):
+            qc.x(0)
+        with expected.if_test((expected.clbits[1], False)):
+            qc.z(1)
+        qc.measure(0, 0)
+        qc.measure(1, 1)
```

Comment:
I just started addressing this deprecation warning and noticed some issues here. I think it should be `qc.z(1)` (missing parenthesis). Also, I think `expected` should be `qc`?